spinegeneric/cli/generate_figure.py

```diff
@@ -368,6 +374,88 @@ def compute_statistics(df):
     return df, stats
 
 
+def output_text(stats, metric):
+    """
+    # Write statistical results into text file
+    :param stats: dict with stat resutls
+    :param metric: currently processed metric (e.g, dti_fa,...)
+    """
+
+    def check_p_value(p_val):
```

Comment:
the main purpose of this function is to format the output p-value (checking it is the "how"). Usually we name functions as "what" instead of "how", so i suggest naming it `format_p_value()`